### PR TITLE
Feature/handle server error.

### PR DESCRIPTION
**Pull request by raquel-ucl**
Do #92 first.

This is trying to capture errors coming from communication with the ORACC server.
Could do with some code review although it is not ready to merge.

### CODE REVIEW

File: python/nammu/SOAPClient/SOAPClient.py [hunk 1]
```diff
@@ -113,4 +127,5 @@ def get_server_logs(self):
             print autolem
             print "@"*30
 
-        return oracc_log, request_log, autolem
\ No newline at end of file
+        return oracc_log, request_log, autolem
+
```
Comment (jenshnielsen) [inline python/nammu/SOAPClient/SOAPClient.py]: Trailing whitespace?

File: python/nammu/SOAPClient/SOAPClient.py [hunk 2]
```diff
@@ -45,7 +50,10 @@ def send(self):
         url = "{}:{}".format(self.url, self.port)
         headers = dict(self.request.get_headers())
         body = self.request.get_body()
-        self.response = requests.post(url, data=body, headers=headers)
+        try:
+            self.response = requests.post(url, data=body, headers=headers)
+        except RequestException:
+            raise
```
Comment (jenshnielsen) [inline python/nammu/SOAPClient/SOAPClient.py]: This can probably be removed

File: python/nammu/controller/NammuController.py [hunk 3]
```diff
@@ -335,45 +340,110 @@ def send_command(self, command, project):
...
-            self.atfAreaController.setAtfAreaText(autolem)
-            self.log("        Lemmatised ATF received from server.\n")
-
-        if oracc_log:
-            validation_errors = self.get_validation_errors(oracc_log)
-            self.atfAreaController.view.error_highlight(validation_errors)
-            self.log("        See highlighted areas in the text for errors and check again.\n\n")
+                              atf_text=nammu_text.encode('utf-8'))
+        
+        self.send_request(client)
```
Comment (jenshnielsen) [inline python/nammu/controller/NammuController.py]: remove?

**Comment by raquel-ucl** [comment]
Code is ready I think, just needs to have some tests in place.

File: python/nammu/SOAPClient/SOAPClient.py (hunk 1 above)
Comment (raquel-ucl) [inline python/nammu/SOAPClient/SOAPClient.py]: gone :)

File: python/nammu/SOAPClient/SOAPClient.py (hunk 2 above)
Comment (raquel-ucl) [inline python/nammu/SOAPClient/SOAPClient.py]: done.

**Comment by raquel-ucl** [comment]
Ready to re-review, @jenshnielsen.

**Pull request merged**